### PR TITLE
[PLAYER-4134] Replay icon

### PR DESCRIPTION
**Pull request by cacel**
[iOS][Android][OoyalaSkinSampleApp] Replay icon is displayed on seeking back after end of video and play instead replay from sec 0.

### CODE REVIEW

**Review by brol1dev — CHANGES_REQUESTED** [review]
There's a conflict in the PR, please fix.

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinBridgeEventHandlerImpl.java [hunk 1]
```diff
@@ -104,6 +107,11 @@ public void handleRewind() {
     _player.seek(playheadTime);
   }
 
+  public void handleReplay() {
+    _player.seek(0);
```
Comment (pilievOoyala) [inline sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinBridgeEventHandlerImpl.java:111]: Is this enough?  What happens to ads? Do they replay using this method?

**Comment by brol1dev** [comment]
@cacel also there's a related issue that we just merged a fix (#745) Can you check that one out too and see how different it is? I'm not sure if there's a difference or if the tickets are actual duplicates.

File: sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinBridgeEventHandlerImpl.java (hunk 1 above)
Comment (cacel) [reply sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinBridgeEventHandlerImpl.java:111]: The seek is set to 0. The ads will be shown but without adMode (same behavior in web).

**Comment by cacel** [comment]
@rcvrgs Both tickets are similar. Basically when the video is finished (with ads or without them) and after scrubbing, the button replay starts from where you finished to scrub. 
This PR, when the endScreen is shown and the user uses the scrubber bar and clicks replay button, the content will start from the beginning.

**Review by akshaydch11 — APPROVED** [review]

**Comment by pilievOoyala** [comment]
@rcvrgs Can you re-review?  I will address the issue about replay in a separate thread.

**Comment by DmitriiMaslov** [comment]
@cacel do we need to move it to DEV branch? @SergeyBicarte  @bychkovdmitry @orobles-ooyala  Please review

**Comment by DmitriiMaslov** [comment]
The related ticket marked as done. PR is too old to merge it into Candidate so I am closing it.

**Pull request closed without merging**